### PR TITLE
update deps

### PR DESCRIPTION
**Pull request by christopherjbaker**

### CODE REVIEW

File: polyfill.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+if (typeof window.Map === 'undefined') {
+	window.Map = require('can-cid/map/map');
+}
+
+if (typeof window.Set === 'undefined') {
+	window.Set = require('can-cid/set/set');
+}
+
+window.requestAnimationFrame = function(callback) {
```
Comment (BigAB) [inline polyfill.js]: Why no check for RAF before assignment? Looks like you'd be destroying RAF for everyone as it is.

Comment (christopherjbaker) [reply polyfill.js]: Because i like to break things

File: package.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
     "viewmodel"
   ],
   "dependencies": {
-    "can-define": "^1.0.25",
+    "can-define": "^1.5.0",
```
Comment (phillipskevin) [inline package.json]: Do you need to add `can-cid` as a dependency?

File: react-view-model.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
+require("./polyfill");
 var assign = require("can-util/js/assign/assign");
 var Component = require("./component");
-var namespace = require('can-namespace');
+var namespace = require("can-namespace");
```
Comment (phillipskevin) [inline react-view-model.js]: it doesn't look like this is a dependency either.

**Pull request merged**